Tests/ImagineEngineTests/Utilities/ImageMockFactory.swift

```diff
@@ -8,8 +8,12 @@ import Foundation
 import CoreGraphics
 @testable import ImagineEngine
 
-final class ImagineMockFactory {
+final class ImageMockFactory {
```

Comment:
Haha, this is the best typo - I was so into the whole "Imagine" thing that I accidentially name the class "Imagine" instead of "Image" 😅 Thanks for fixing! 👍

Reply:
That's the best typo indeed 😄